components/Banner/src/MDCBannerView.m

```diff
@@ -375,13 +375,23 @@ - (CGSize)sizeThatFits:(CGSize)size {
   return CGSizeMake(size.width, frameHeight);
 }
 
+- (CGSize)intrinsicContentSize {
+  CGFloat intrinsicContentHeight = [self sizeThatFits:self.bounds.size].height;
+  return CGSizeMake(UIViewNoIntrinsicMetric, intrinsicContentHeight);
+}
+
 - (void)updateConstraints {
   MDCBannerViewLayoutMode layoutMode = [self layoutModeForSizeToFit:self.bounds.size];
   [self updateConstraintsWithLayoutMode:layoutMode];
 
   [super updateConstraints];
 }
 
+- (void)layoutSubviews {
+  [super layoutSubviews];
+  [self invalidateIntrinsicContentSize];
```

Comment:
This doesn’t seem like something we should be doing....can we remove this? It is similarly self-referential and feels like it could result in an infinite loop.

Reply:
I totally understand the concern here. Let's follow up offline on this PR:)